modules/appeals_api/app/controllers/appeals_api/v2/decision_reviews/higher_level_reviews_controller.rb

```diff
@@ -112,7 +115,8 @@ def new_higher_level_review
       auth_headers: request_headers,
       form_data: @json_body,
       source: request_headers['X-Consumer-Username'],
-      api_version: 'V2'
+      api_version: 'V2',
+      veteran_icn: request_headers['X-VA-ICN']
```

Comment:
A couple ideas involving whitespace in the headers: 

1. We could add a `.presence&.strip` onto here just in case the headers are passed in as only or have extra whitespace. We'll still have access to the raw value with `auth_headers` if we need it.
2. We update our header schema to `$ref: non_blank_string.json` and let the schema validator error if only whitespace is detected (which means we'd probably still want to `.strip` the header value when storing onto the model)
3. We add a pattern to the schema to only allow alphanumeric characters.

Reply:
Went with #1